meeting-notes/lcwg-dev-58.md

```diff
@@ -0,0 +1,60 @@
+
+
+### ZWCD Updates
+Pacu is working in darksidewalletd benchmarks for Zingo that will be extended and made available to other wallets. Also reviewing PRs from NH and ECC. 
+
+He requested suggestions on improving regtest performance because the testcase generation was taking too much time and had to move on to a different approach.
+
+For this Kris pointed out [this code](https://github.com/zcash/zcash/blob/master/qa/rpc-tests/test_framework/util.py) that could help set up cached regtest states and then use the resulting wallet directory to generate the darksidewalletd datasets.
+
+In terms of organizing LCWG Pacu and Decentralistdan have been working to get the Zoom call venue ready for today's call. Additionally Pacu is reaching out to all development teams that had previously participated on LCWG to invite them over and get feedback and their preferred times in an attempt to accomodate everyone better. 
+
+
+### ECC Updates
+
+#### SDKs pre-release testing 
+
+Kris mentioned that testing of the SDKs would be appreciated, clients of the sdk should check [zcash-wallet-dag](https://zcash.github.io/developers/zcash-wallet-dag.svg)
+
+Zooko: announcing taking on roles EM and Product for SDKs
+on [this post on the forums](https://forum.zcashcommunity.com/t/all-ecc-teams-focused-on-wallet-performance/42860/107?u=zooko)
+
+ECC plans to ship SbS capable SDKs once Str4d is back. No other changes will be done in terms of mitigating mempool congestions issues. 
+
+LCWG discord channel users wanting to join the Zashi beta testing program might message Zooko
+
+
+#### Darksidewalletd Testing and Sbs 
+Kris: There are Darksidetests that need review and fixing on the SDKs on [This PR](https://github.com/zcash/ZcashLightClientKit/pull/1191). Pacu will review and prioritize any work if possible since there are other tasks in his backlog.
+
+Kris also suggested that the Dlwd tests have a definition language so that tests can be written once and ran on whichever wallet and those tests suites can be user on z.cash to actually get it listed on the official wallet list on the website.
+
+https://github.com/zcash/ZcashLightClientKit/pull/1191
+
+
+###  Action Items from previous meeting
+- ~Enable ZIP-317 booleans on the sdk and do small releases~. This has been deprioritized by ECC to focus on Sbs. Minimum fee of 10000 zat may or may not be included in next release.
```

Comment:
The ECC team discussed this after the call and verified that the next version of each of the SDKs will ship with a 10,000 ZAT fixed fee.

Reply:
Thanks for clarifying this! I'll will include in the notes as a follow up